mobile/src/modules/auth/components/passcode/EnterPasscodeModal.tsx

```diff
@@ -15,6 +15,7 @@ interface OwnProps {
   onSuccess: () => void;
   onCancel?: () => void;
   onDismiss?: () => void;
+  onReset?: () => void;
```

Comment:
if you pass a default func, which is perfect, you won't need further Null checks (see below)